content/v2.0/views/views-and-exposures.md

```diff
@@ -108,3 +104,55 @@ module Bookshelf
   end
 end
 ```
+
+### Depending on other exposures
+
+Sometimes one exposure will depend on value of another. You can depend on another exposure by naming it as a positional argument for your exposure block.
+
+```ruby
+# app/views/books/show.rb
+
+module Bookshelf
+  module Views
+    module Books
+      class Show < Bookshelf::View
+        include Deps[
+          "repositories.book_repo",
+          "repositories.author_repo"
+        ]
+
+        expose :book do |id:|
+          book_repo.get!(id)
+        end
+
+        expose :author do |book|
```

Comment:
:thought_balloon: Is is possible to define multiple positional parameters for dependent exposures? I assume multiple followed by keyword parameters as needed?